temoa/model_checking/unit_checking/screener.py

```
@@ -0,0 +1,321 @@
+    # wrap it up
+    if report_dir:
+        _write_report(report_dir, report_entries)
+    logger.info('Finished Units Check')
+    return all_clear
+
+
+def _write_report(report_dir: Path, report_entries: list[str]) -> None:
+    """write out a report if the path is specified"""
+    from datetime import datetime
+
+    timestamp = datetime.now().strftime('%Y-%m-%d_%H%M%S')
```

Ruff (0.14.8)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)